sequencer/src/lib.rs

```diff
@@ -846,11 +846,6 @@ pub mod testing {
             self
         }
 
-        pub fn epoch_start_block(mut self, epoch_start_block: u64) -> Self {
-            self.config.epoch_start_block = epoch_start_block;
-            self
-        }
```

Comment:
if we remove this, shouldn't we also remove `epoch_height()` above, we might as well specify all of these via env var.

Reply:
ah nvm, this is unrelated to this PR, you can ignore it.